kubectl-fdb/cmd/analyze.go

```diff
@@ -70,6 +70,11 @@ func newAnalyzeCmd(streams genericclioptions.IOStreams) *cobra.Command {
 				return err
 			}
 
+			ignoreConditions, err := cmd.Flags().GetStringArray("ignore-condition")
```

Comment:
Should we add a validation to the arguments such that it's within ProcessGroupConditionType.AllProcessGroupConditionTypes()?

Reply:
I think that's a good idea and we should fail early if a condition is passed that is unknown.